Index.html

```diff
@@ -38,17 +38,209 @@ <h2>
                 a project you need coded, don't hesitate
                 to contact me.
             </p>
-
+         <div class="icon-link">
             <ul>
                 <li><a href="#"><span class="fa fa-twitter"></span></a></li>
                 <li><a href="#"><span class="fa fa-angellist"></span></a></li>
                 <li><a href="#"><span class="fa fa-github"></span></a></li>
                 <li><a href="#"><span class="fa fa-linkedin"></span></a></li>
             </ul>
         </div>
+        </div>
+        </section>
+
+        <section class="Work">
+            
+            <h1>My Recent Work</h1>
+            <div class="grid-items">
+                <div class="flex-box">
+                    <img src="./Images/SecondPortfolio.svg" alt="My-Work">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+
+                    <div class="list-box">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                    <div class="See-btn">
+                        
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+                </div>
+
+                <div class="flex-box">
+                    <img src="./Images/SecondPortfolio.svg" alt="My-Work">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+
+                    <div class="list-box">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                  
+                    <div class="See-btn">     
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+
+                </div>
```

Comment:
- [ ] As I've mentioned in the summary, kindly remove the white space after the closing `<div>` tag on line 89 and other places.
- [ ] Kindly ensure that the spacings to the left and right of each project technologies are equal. See the difference below.

| Figma Design | Your Design |
| ------------------ | ----------------- |
| ![original](https://user-images.githubusercontent.com/26856491/121516779-853d9a00-c9e6-11eb-8004-a499c47129cb.png) | ![clone](https://user-images.githubusercontent.com/26856491/121516799-8a024e00-c9e6-11eb-9698-4b4ccada37c3.png) |